bbl_screen-patch/interpose.cpp

```diff
@@ -574,6 +597,16 @@ SWIZZLE(ssize_t, write, int fd, const void *p, size_t n)
         } else {
             printf("backlight off\n");
         }
+    } else if (fd == chamber_led_fd) {
+        printf("interposed write() on backlight, ");
+        if (*(char*)p != '0') {
+            printf("writing %d instead\n", native.chamberledSetting);
+            std::string valueText = std::to_string(native.chamberledSetting);
+            next(fd, valueText.c_str(), valueText.length());
+            return n;
```

Comment:
This appears to "AND" the chamber LED with X1Plus.  Do we want to override it both directions?  Or just as an AND?